src/main/java/tasks/yura/Task3.java

```diff
@@ -1,41 +1,34 @@
 package tasks.yura;
 
-import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class Task3 {
     public static void main(String[] args) {
 
-        int[] array = {4,8,16,3,5,42,4,5,34,78,41,4,38,87,3}; // created random array for example
-        //конвертнув арей в арей ліст
-        ArrayList<Integer> myList = new ArrayList<>();
-        for (int i = 0; i < array.length; i++){
-            myList.add(array[i]);
-        }
+        List<Integer> myList = Arrays.asList(4,8,16,3,5,42,4,5,34,78,41,4,38,87,3); // created random array for example
 
-        // знайшов мінімальне значення
+        // found min
         int min = myList.get(0);
         for (int i = 1; i < myList.size() ; i++) {
             if(min> myList.get(i)){
                 min = myList.get(i);
             }
         }
-
-        //знайшов середнє арифметичне всього масиву
+        //found average
         int sum = 0;
         for (int i : myList) {
             sum += i;
         }
         int average = sum / myList.size();
 
```

Comment:
During our training we are going to use java 8.
Please start learning it.
Now change lines 19-23 to
//found average
        double avg = myList.stream()
                .mapToDouble(i -> i)
                .average()
                .getAsDouble();